packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/main/index.md

```diff
@@ -106,7 +106,7 @@ When users initiate a password recovery, they are sent an email based on the **F
       | otp            | *${oneTimePassword}*    |
       | state          | *${request.relayState}* |
 
-      After your updates the URL should take the following format: `http://[your app's address]?otp=${oneTimePassword}&state=${request.relayState}`.
+      After your updates the URL should take the following format: `http://${yourOktaDomain}?otp=${oneTimePassword}&state=${request.relayState}`.
```

Comment:
Have deleted comma, changed to "The URL should have the following format". 

Not sure why "should" is indicating passive voice in this example.

Reply:
If not passive, then words and phrases such as "should", "could", "should have", "could have", etc. are simply extra words that add no value. Either something is or is not the case, either the URL is updated or it isn't. If the case not not updating is an error, then call that out, otherwise be declaritive instead of hedging:

> After your update, the format of the URL is: ...

OR

> After updating, check that the format of the URL is correct:
> ...
> If not...

Reply:
@mauricesharp-okta Have changed to 

> After your changes, the URL takes the form `http://${yourAppDomain}?otp=${oneTimePassword}&state=${request.relayState}` where `${yourAppDomain}` is the domain of your application.

Reply:
I don't see that change in the current commits though in re-reading the surrounding context I'm not sure that this sentence is required. I may be missing something, but it appears that this is a section about modifying the template for the magic link email by providing a custom URL instead of the default one (`resetPasswordLink`), or possibly changing the value of that symbol.

Basically, what's the relevance of that URL? Is it text that appears in the email template? If so, where does it appear? Does it relate to the code snippet shown in step 6? If so, how? Perhaps show a modified code snippet?

Reply:
Still don't see changes, and it still reads odd (the URL takes the form). Also, why is this after some action? Would'nt it be better to includes this all in whatever action that's being documenting. For example:

Change the foo-bar URL to the following form, substituting the domain of your applicaiton for `${yourAppDomain}`:
`http://${yourAppDomain}?otp=${oneTimePassword}&state=${request.relayState}`

Also note, are oneTimePassword and request.relayState also variables such as yourAppDomainName, or is this statement using the same syntactical element (`${variableName}`) for two completely different purposes?